charts/newrelic-infrastructure-v3/templates/NOTES.txt

```diff
@@ -4,13 +4,112 @@ Your deployment of the New Relic Infrastructure agent is complete if you have no
+and kubelet.affinity.nodeAffinity to customize nodeAffinity.
+{{- end }}
+
+{{- if .Values.resources }}
+
+-> WARNING LEGACY CONFIG <-
+You have specified the legacy "resources" option into values, however the new version of the chart
+supports different resources per component, therefore the value specified has been ignored.
+
+Please use ksm.resources, controlPlane.resources and kubelet.resources.
+{{- end }}
+
+{{- if .Values.tolerations }}
+
+-> WARNING LEGACY CONFIG <-
+You have specified the legacy "tolerations" options into values, however the new version of the chart
+supports different tolerations per component, therefore the value specified has been ignored.
+
+Please use ksm.tolerations, controlPlane.tolerations and kubelet.tolerations.
+{{- end }}
+
+{{- if .Values.image }}
+
+-> WARNING LEGACY CONFIG <-
+You have specified into values one of the legacy options "image.*".
+The following values are no longer supported and are currently ignored.
+ - image.repository
+ - image.tag
+ - image.pullPolicy
+ - image.pullSecrets
+
+Notice that the 3.x version of the integration uses 3 different images.
+Please set:
+ - images.forwarder.* to configure the image in charge of sending data to newrelic backend
+ - images.agent.* to configure the image bundling the agent and onHost integration
+ - images.integration.* to configure the image in charge of scraping k8s data
+{{- end }}
+
+{{- if (or .Values.enableWindows .Values.windowsOsList .Values.windowsSecurityContext .Values.windowsNodeSelector)}}
+
+-> WARNING LEGACY CONFIG <-
+Windows is currently supported by 2.x charts only and therefore "windowsOsList" "windowsSecurityContext" "windowsNodeSelector"
+are no longer supported.
+
+{{- end }}
+
+{{- if ( or .Values.controllerManagerEndpointUrl  .Values.schedulerEndpointUrl .Values.etcdEndpointUrl .Values.apiServerEndpointUrl )}}
+
+-> WARNING LEGACY CONFIG <-
+Values "controllerManagerEndpointUrl", "etcdEndpointUrl", "apiServerEndpointUrl", "schedulerEndpointUrl" are no longer
+supported, please specify them in the config file. You can add them in the autodiscovery section. Example for etcdEndpointUrl:
+
+#  autodiscover:
+#    - selector: "tier=control-plane,component=etcd"
+#      namespace: kube-system
+#      matchNode: true
+#      endpoints:
+#        - url: https://localhost:4001
+#          insecureSkipVerify: true
+#          auth:
+#            type: bearer
+
+{{- end }}
+
+{{- if ( or .Values.etcdTlsSecretName .Values.etcdTlsSecretNamespace )}}
+
+-> WARNING LEGACY CONFIG <-
+Values "etcdTlsSecretName" and "etcdTlsSecretNamespace" are no longer supported, please specify them in the config
+file. Example:
+# - endpoints:
+#     - url: https://localhost:9979
+#       insecureSkipVerify: true
+#       auth:
+#         type: mTLS
+#         mtls:
+#           secretName: secret-name
+#           secretNamespace: secret-namespace
+{{- end }}
```

Comment:
`{{- fail "Please migrate your control plane config to the nri-kubernetes v3 format " }}`